README.md

```diff
@@ -44,6 +44,26 @@ override func scrollViewDidEndDragging(scrollView: UIScrollView, willDecelerate
 }
 ```
 
+Implement the SectionScrubberDelegate (all functions have a default implementation and thus are optional):
+
+```swift
+extension RemoteCollectionController: SectionScrubberDelegate {
+    func sectionScrubber(sectionScrubber: SectionScrubber, didRequestToSetContentViewToYPosition yPosition: CGFloat) {
+        // function is called when collectionView should update it's contentOffset to respond to dragging of the 
```

Comment:
Uppercase in function, although `Called when collectionView....` might be better.